collector/mssql.go

```diff
@@ -19,18 +19,53 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
+	"regexp"
+	"sync"
+	"time"
 
 	"github.com/StackExchange/wmi"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"golang.org/x/sys/windows/registry"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
-type sqlInstancesType map[string]string
+var (
+	mssqlCollectorWhitelist = kingpin.Flag(
+		"collector.mssql.class-whitelist",
+		"Regexp of mssql WMI classes to whitelist. Name must both match whitelist and not match blacklist to be included.",
+	).Default(".+").String()
```

Comment:
Would you consider regexp better than comma-separated list here? Since the exact allowed names are known, regexp doesn't seem to add much? (You can even use the `Enums` kingpin type then to get validation early)

Reply:
all the other white/blacklist options (iis, nic, logicaldisk) are using regex, so I'd figure I'd keep things consistent.   

But I see your point.  

I'll mimic the `collectors.enabled` option instead.